prescriptions/te_/tensorflow/tf_protobuf4210rc1.yaml

```
@@ -0,0 +1,22 @@
+units:
+  steps:
+  - name: TensorflowProtobuf4x21x0rc1Step
+    type: step.Group
+    should_include:
+      adviser_pipeline: true
+    match:
+    - group:
+      - package_version:
+          name: tensorflow
+          index_url: https://pypi.org/simple
+      - package_version:
+          name: protobuf
+          version: ==4.21.0-rc1
```

I think this has to be a normalized version:

```suggestion
          version: ==4.21.0rc1
```

---

Thanks! Fixed in https://github.com/thoth-station/prescriptions/pull/23483/commits/f169c0b4907e8b300db27339d9deb1de24e3292d